src/frontend/js/hooks/useDashboardAddressForm.tsx

```diff
@@ -108,7 +108,7 @@ export const useDashboardAddressForm = (address?: Address) => {
       />
       <CountrySelectField
         aria-invalid={!!formState.errors.country}
-        aria-required={false}
+        aria-required={true}
```

Comment:
Considering my last comment IMO `aria-required` would not be needed anymore because `required` will be enough and seems to be supported by screen readers nowadays.

Reply:
Yup, indeed !